### PR TITLE
Bump tonic version to avoid vulnerability

### PR DESCRIPTION
**Pull request by Laursen79**

### CODE REVIEW

**Review by Mikkel-Boje — APPROVED** [review]

**Comment by t-lohse** [comment]
I actually need this vulnerability, pls don't merge this

**Review by UnluckyBird — APPROVED** [review]

**Comment by t-lohse** [comment]
@Laursen79 stop them!
image

File: client/Cargo.toml [hunk 1]
```diff
@@ -16,4 +16,4 @@ uuid = { version = "1.10.0", features = ["v4", "fast-rng", "macro-diagnostics"]
 anyhow = "1.0.89"
 
 [build-dependencies]
-tonic-build = "0.12.2"
+tonic-build = "0.12.3"
```
Comment (t-lohse) [inline client/Cargo.toml:19]:
```suggestion
tonic-build = "0.12.2"
```

**Review by t-lohse — CHANGES_REQUESTED** [review]

File: common/Cargo.toml [hunk 2]
```diff
@@ -5,8 +5,8 @@ edition = "2021"
 
 [dependencies]
 prost = "0.13.3"
-tonic = "0.12.2"
+tonic = "0.12.3"
```
Comment (t-lohse) [inline common/Cargo.toml:8]:
```suggestion
tonic = "0.12.2"
```

File: common/Cargo.toml [hunk 3]
```diff
@@ -5,8 +5,8 @@ edition = "2021"
...
 serde = { version = "1.0.210", features = ["derive"] }
 
 [build-dependencies]
-tonic-build = "0.12.2"
+tonic-build = "0.12.3"
```
Comment (t-lohse) [inline common/Cargo.toml:12]:
```suggestion
tonic-build = "0.12.2"
```

File: server/Cargo.toml [hunk 4]
```diff
@@ -13,4 +13,4 @@ axum = "0.7.6"
 serde = { version = "1.0.210", features = ["derive"] }
 
 [build-dependencies]
-tonic-build = "0.12.2"
+tonic-build = "0.12.3"
```
Comment (t-lohse) [inline server/Cargo.toml:16]:
```suggestion
tonic-build = "0.12.2"
```

**Review by t-lohse — CHANGES_REQUESTED** [review]
plz

**Pull request merged**